plugin/trino-hive-hadoop2/bin/common.sh

```diff
@@ -76,12 +76,30 @@ function cleanup_hadoop_docker_containers() {
     cleanup_docker_containers "${DOCKER_COMPOSE_LOCATION}"
 }
 
-function termination_handler(){
+function termination_handler() {
     set +e
     cleanup_docker_containers "$@"
     exit 130
 }
 
+# Check that all arguments are the names of non-empty variables.
+function require_env() {
```

Comment:
`is_variable_non_empty`

"require_..." would be appropriate if the function did `exit 1` when the condition is not met.

Reply:
Maybe `check_vars`? Or I could actually make it exit.

The unwritten assumption is that the caller had `set -e`, so the `return 1` here is effectively an `exit 1`.

Reply:
it's normal for a function to communicate via return status. it's up to the caller to use this information by means of an `if`, `||` or `set -e`.

> Maybe check_vars?

works for me.